### PR TITLE
build: Package for Conda

### LINKED ISSUE #54: Package for Conda
I saw #41 is pending yet, but I think it might be interesting to have this package in Conda too, because it is used quite a lot, especially in the scientific-linked open data environment. Building a conda doesn't seem very hard. I have never build one, but I could try it. There are many [documentation](https://conda.io/docs/user-guide/tasks/build-packages/index.html) about it.

What do you think?

### PR DESCRIPTION
**Pull request by aadarsh-ram**
**Description**
The openfoodfacts-python repo has only a PyPI release. It would be helpful to have a Conda package too.
This requires an Anaconda API token (ANACONDA_TOKEN) added in the secrets of the repository.

**Related issue(s)**
- Fixes #54 

**Part of**
- #76 

**Solution**
- GitHub Action for building a Conda package on new release creation (publish-conda.yml)
- Created a meta.yml file

### CODE REVIEW

**Comment by alexgarel** [comment]
Seems really cool @aadarsh-ram

File: .github/workflows/publish-conda.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout repo
+      uses: actions/checkout@v1
+    - name: publish-to-conda
+      uses: MichaelsJP/conda-package-publish-action@v1.0.0
+      with:
+        subDir: 'conda'
+        AnacondaToken: ${{ secrets.ANACONDA_TOKEN }}
```
Comment (alexgarel) [inline .github/workflows/publish-conda.yml:18]: I added a secret with a  token I got from https://anaconda.org/.

**Review by alexgarel — COMMENTED** [review]
@aadarsh-ram that's cool, but I think, I have to make a PR my own to be able to test (but when tested I will come back to your PR to validate)

File: .github/workflows/publish-conda.yml (hunk 1 above)
Comment (aadarsh-ram) [reply .github/workflows/publish-conda.yml:18]: @alexgarel Great, I already tested it with my fork and it seems to build and upload!

Comment (aadarsh-ram) [reply .github/workflows/publish-conda.yml:18]: So, on a new release creation - it should automatically upload to PyPI and conda as well

**Comment by alexgarel** [comment]
Also the proposal is really good @aadarsh-ram I may not work on this immediatly because I have some more urgent task right now. But thanks really.

**Comment by aadarsh-ram** [comment]
@alexgarel Glad to hear that! No issues, you can complete your work and come back to this afterwards. Thanks!

**Comment by alexgarel** [comment]
@aadarsh-ram I decided to merge this PR, we will see if it fails on next release ;-)

**Comment by aadarsh-ram** [comment]
> @aadarsh-ram I decided to merge this PR, we will see if it fails on next release ;-)

Cool!

**Pull request merged**